Phase1ScanHistoMaker.cc

```diff
@@ -80,8 +80,8 @@
 //#define COMPLETE 1
 #define EXPRESS 1
 
-#define HV_Scan     1
-#define Timing_Scan 0
+#define HV_Scan     0
+#define Timing_Scan 1
```

Comment:
Well, these two lines I would not modify so that PR#2 remains a HV scan related. However, I am now noticing that PR#2 actually touches these lines and makes the HV scan the default scan. Perhaps we need to control the scan type through a compiler flag instead of having it hardcoded.